src/colvartypes.h

```diff
@@ -11,6 +11,7 @@
 #define COLVARTYPES_H
 
 #include <vector>
+#include <array>
```

Comment:
Not sure why this is included.

Reply:
I tried `std::array` but ended up with simple C arrays, so it was a left-over. I have removed it in the latest commit.